.devcontainer/Dockerfile

```diff
@@ -6,12 +6,14 @@ FROM mcr.microsoft.com/vscode/devcontainers/ruby:0-${VARIANT}
 ARG NODE_VERSION="none"
 RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
 
-# [Optional] Uncomment this section to install additional OS packages.
-# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-#     && apt-get -y install --no-install-recommends <your-package-list-here>
+# Install additional OS packages.
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends fish \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Since devcontainers are used locally as a shell, I don't believe this best practice to limit container layer size is necessary (a devcontainer user may apt install new packages from shell).

Reply:
I added this because CodeFactor complained without it, but will add a suppression comment instead

Reply:
Ah, I see.  I wonder if we can exclude the entire .devcontainer directory from CodeFactor, since its not really our application code anyhow.

Reply:
OK, I've force-updated to remove these lines. We could add `# hadolint ignore=DL3009` to the Dockerfile, but it looks like if you'd rather exclude the entire directory you need to do that through the CodeFactor settings. (https://support.codefactor.io/i15-ignoring-code-files)

Reply:
Thanks for the reference! It's kinda weird code factor doesn't allow configuring the ignore list file a file in repository, so for reference this is what I put in their web settings for ignore list for this repo for now:
```
.vscode/**
.devcontainer/**
```

Reply:
Ugh, I don't know why this isn't picking up when I put it in the CodeFactor web settings, I'll just add the manual exclusion you mentioned for now.